test_utils/__init__.py

```diff
@@ -1,10 +1,73 @@
 """
 Test utilities.
+"""
 
-Since pytest discourages putting __init__.py into testdirectory
-(i.e. making tests a package) one cannot import from anywhere
-under tests folder. However, some utility classes/methods might be useful
-in multiple test modules (i.e. factoryboy factories, base test classes).
+from rest_framework.test import APIClient, APITestCase
 
-So this package is the place to put them.
-"""
+from test_utils.factories import UserFactory
+from test_utils.mixins import JwtMixin
+
+TEST_USERNAME = "api_worker"
+TEST_EMAIL = "test@email.com"
+TEST_PASSWORD = "QWERTY"
+TEST_LMS_USER_ID = 5
+
+
+class ProgramIntentAPITestCase(JwtMixin, APITestCase):
+    """
+    Base class for API Tests
+    """
+
+    def setUp(self):
+        """
+        Perform operations common to all tests.
+        """
+        super().setUp()
+        self.create_user(
+            username=TEST_USERNAME,
+            email=TEST_EMAIL,
+            password=TEST_PASSWORD,
+            is_staff=True,
+        )
+        self.client = APIClient()
+        self.client.login(username=TEST_USERNAME, password=TEST_PASSWORD)
+
+        # TODO me and Varsha will write our own setups and any overlapping can be put in here
```

Comment:
Just a gentle reminder not to forget to remove this once you're ready to merge this!